coderabbit.yaml

```
@@ -0,0 +1,275 @@
+# yaml-language-server: $schema=https://coderabbit.ai/integrations/schema.v2.json
+language: es-ES
+tone_instructions: >
+  Usa un tono claro y directo. Señala problemas específicos y ofrece soluciones concretas. 
```

yamllint

[error] trailing spaces

(trailing-spaces)